src/components/dailyQuestsPage/DailyQuests.jsx

```diff
@@ -0,0 +1,244 @@
+import React, { useEffect, useState } from "react";
+import { format, addDays } from "date-fns";
+import QuestModal from "./QuestModal";
+import NavbarTopLoggedIn from "../chooseNewResopage/NavbarTopLoggedIn";
+import SuggestedQuests from "../dailyQuestsPage/SuggestedQuests";
+import WeekDaysCarousel from "./WeekDaysCarousel";
+import QuestList from "./QuestList";
+import AddQuestButton from "./AddQuestButton";
+
+const daysOfWeek = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
+
+const normalizeFrequency = (freq) => {
```

Comment:
extract it in separate file. you can create folder here, helpers or utils and store it there